src/sentry/api/endpoints/organization_teams.py

```diff
@@ -142,25 +143,53 @@ def post(self, request: Request, organization, **kwargs) -> Response:
 
         :pparam string organization_slug: the slug of the organization the
                                           team should be created for.
-        :param string name: the optional name of the team.
-        :param string slug: the optional slug for this team.  If
-                            not provided it will be auto generated from the
-                            name.
+        :qparam string name: the optional name of the team.
+        :qparam string slug: the optional slug for this team. If not provided it will be auto
+                             generated from the name.
+        :qparam bool set_admin: If this is true, the user is added to the as a Team Admin
+                                instead of regular member
         :auth: required
         """
         serializer = TeamPostSerializer(data=request.data)
 
-        if serializer.is_valid():
```

Comment:
Changed from if-else to early return so the diffs between the files are a little difficult to read. I created a [Github Gist here](https://gist.github.com/schew2381/1a727d6647fd2fbd0dd16296c41186a9) if you anyone wants to easily view the new and original POST reqs